orga/tests/integration/components/routes/authenticated/campaigns/details-item-test.js

```diff
@@ -0,0 +1,48 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { run } from '@ember/runloop';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | routes/authenticated/campaign | details-item', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it should display campaign details', async function(assert) {
+    // given
+    let store = this.owner.lookup('service:store');
+    const campaign = run(() => store.createRecord('campaign', {
+      id: 1,
+      name: 'campagne 1',
+    }));
+
+    this.set('campaign', campaign);
+
+    // when
+    await render(hbs`{{routes/authenticated/campaigns/details-item campaign=campaign}}`);
+
+    // then
+    assert.dom('.campaign-details-header__title').hasText('campagne 1');
+  });
+
+  test('it should display target profile related to campaign', async function(assert) {
+    // given
+    let store = this.owner.lookup('service:store');
```

Comment:
Possibilité de mettre le given dans un before ?

Reply:
Après quelques essais, on a vu que ce n'était pas possible de mettre un `this.owner.lookup('service:store')` dans un beforeEach. Cela semble être la norme dans les autres fichiers.

Reply:
La déclaration doit se faire comme suit pour que cela fonctionne 
``` javascript
  let store;

  hooks.beforeEach(function() {
    run(() => {
      store = this.owner.lookup('service:store');
    });
  });
```